src/includes/getting-started-install/javascript.nextjs.mdx

```diff
@@ -0,0 +1,7 @@
+```bash {tabTitle:npm}
+npm install --save @next/plugin-sentry @next/sentry-source-maps
```

Comment:
save is here because we still keep it everywhere in docs - future item will be to remove if determined never applicable